packages/ozone/tests/blob-divert.test.ts

```diff
@@ -0,0 +1,101 @@
+import { SeedClient, TestNetwork, basicSeed } from '@atproto/dev-env'
+import AtpAgent from '@atproto/api'
+import nock from 'nock'
+
+const BLOB_DIVERT_SERVICE_HOST = 'http://blob-report.com'
+const BLOB_DIVERT_SERVICE_PATH = '/xrpc/com.atproto.unspecced.reportBlob'
+const BLOB_DIVERT_SERVICE_AUTH_TOKEN = 'Basic test'
+
+describe('blob divert', () => {
+  let network: TestNetwork
+  let agent: AtpAgent
+  let sc: SeedClient
+
+  beforeAll(async () => {
+    network = await TestNetwork.create({
+      dbPostgresSchema: 'ozone_blob_divert_test',
+      ozone: {
+        blobReportServiceUrl: `${BLOB_DIVERT_SERVICE_HOST}${BLOB_DIVERT_SERVICE_PATH}`,
+        blobReportServiceAuthToken: BLOB_DIVERT_SERVICE_AUTH_TOKEN,
+      },
+    })
+    agent = network.pds.getClient()
+    sc = network.getSeedClient()
+    await basicSeed(sc)
+    await network.processAll()
+  })
+
+  afterAll(async () => {
+    await network.close()
+  })
+
+  afterEach(() => {
+    nock.cleanAll()
+  })
+
+  const mockReportServiceResponse = (
+    status: number,
+    data: Record<string, boolean>,
+  ) =>
+    nock(BLOB_DIVERT_SERVICE_HOST)
```

Comment:
We don't use this pattern of intercepting http reqs anywhere in the codebase so wondering if it's ok to introduce this here?

Reply:
I often avoid these kinds of "live" mocking libraries because they tend to create some weird interactions.  I think we can give it a whirl and if it shows any signs of weirdness, not have any qualms about ripping it out.

It seems like we could pretty easily mock it in the application by placing a method on `BlobDiverter` responsible for getting this response, and instead just mock-out that method.  Node now comes with pretty awesome built-in mocking which makes this quite easy to do: https://nodejs.org/api/test.html#class-mockfunctioncontext

Reply:
cool ended up using the jest.fn mocks.